packages/mobile/src/components/usdc-manual-transfer-drawer/USDCManualTransferDrawer.tsx

```diff
@@ -97,10 +109,24 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
 
 export const USDCManualTransferDrawer = () => {
   const styles = useStyles()
+  const dispatch = useDispatch()
   const { toast } = useToast()
-  const { isOpen, onClose, onClosed } = useUSDCManualTransferModal()
+  const { isOpen, onClose, onClosed, data } = useUSDCManualTransferModal()
+  const { source, amount: amountInCents, startPurchaseParams } = data ?? {}
   const { onPress: onPressLearnMore } = useLink(USDCLearnMore)
-  const neutral = useColor('neutral')
+  const { neutral, specialLightGreen } = useThemeColors()
+
+  const stage = useSelector(getPurchaseContentFlowStage)
+  const error = useSelector(getPurchaseContentError)
+  const isUnlocking = !error && isContentPurchaseInProgress(stage)
+  const { data: balanceBN } = useUSDCBalance({
```

Comment:
just to check - is anyone in the parent path for this already polling? we should probably dedupe this somewhere if so. not sure i love this pattern (I helped create)

Reply:
hm seems like there are actually multiple pollers at this point even before opening the manual transfer drawer 👀 should do a pass to dedup.

Reply:
ill add ticket